source/server/http/admin.cc

```diff
@@ -425,5 +425,13 @@ const Network::Address::Instance& AdminImpl::localAddress() {
   return *server_.localInfo().address();
 }
 
+void AdminImpl::removeHandler(const std::string& prefix) {
+  auto it = std::find_if(handlers_.begin(), handlers_.end(),
```

Comment:
I tried using remove_if, but it turns out that the way it removes is by moving members. So because of the const members in the struct I could not use remove_if. Let me know if you have a better solution. I dug around a little bit, but nothing looked better to me that finding and erasing.